cmd/secrets-store-csi-driver/main.go

```diff
@@ -41,9 +42,7 @@ var (
 	endpoint           = flag.String("endpoint", "unix://tmp/csi.sock", "CSI endpoint")
 	driverName         = flag.String("drivername", "secrets-store.csi.k8s.io", "name of the driver")
 	nodeID             = flag.String("nodeid", "", "node id")
-	debug              = flag.Bool("debug", false, "sets log to debug level")
```

Comment:
Do we want to introduce a breaking change here or should we return a warning that this as deprecated is provided?

Reply:
Done!